### PR TITLE
Fix/discovery page bugs heal

### PR DESCRIPTION
**Pull request by ZakirG**
Jira Tickets: [PXP-396](https://ctds-planx.atlassian.net/browse/HP-396), [HP-393](https://ctds-planx.atlassian.net/browse/HP-393), [HP-463](https://ctds-planx.atlassian.net/browse/HP-463)

This PR: 
- Adds a gitops.json attribute inputSubtitle, a string to be displayed underneath the Discovery page search box. We'll use this attribute to show examples of possible searches.
- Removes the sort icon from the Data Availability column header
- Alters the filter options in the Data Availability column header

### Improvements
- New configurable subtitle helper text under the Discovery page search box in the gitops.json attribute inputSubtitle.
- UX improvements in the Discovery page's Data Availability column header -- 3 filter options, remove sort icon
- Removing `AccessLevel.UNACCESSIBLE` from table sorting and filtering options

### CODE REVIEW

File: src/Discovery/DiscoveryMDSSearch.tsx
Comment (mfshao) [inline src/Discovery/DiscoveryMDSSearch.tsx:18]: why don't put this as part of the `placeholder` msg for `Input`?

Comment (ZakirG) [reply src/Discovery/DiscoveryMDSSearch.tsx:18]: That could be an alternative way to do this, for sure. But the ticket asked to put it below the box, I confirmed with PM Noah.

Comment (ZakirG) [reply src/Discovery/DiscoveryMDSSearch.tsx:18]: (and, I'll wait until feat/download PR is merged before I merge this. I'll be pushing more UI tickets into this PR)

**Review by nmetokishlubsky — APPROVED** [review]

**Review by nmetokishlubsky — APPROVED** [review]

File: src/Discovery/Discovery.tsx
Comment (mfshao) [inline src/Discovery/Discovery.tsx:375]: To double check, you do have `AccessLevel.UNACCESSIBLE` for data, but you don't want to have a filter for it in the table? How would those data show up in table?

Comment (ZakirG) [reply src/Discovery/Discovery.tsx:375]: That's right. Those data will show up in the table with no row value in the Data Availability column. We're de-emphasizing the representation of locked / unaccessible data in the UI.

Comment (ZakirG) [reply src/Discovery/Discovery.tsx:375]: I have this branch deployed in qa-heal ~ https://qa-heal.planx-pla.net/

Comment (mfshao) [reply src/Discovery/Discovery.tsx:375]: I personally think this is a interesting UX, and probably is more HEAL specific so likely it will be needed to be more customizable

Can you explicitly mention it (removing `AccessLevel.UNACCESSIBLE` from table sorting options) in `Improvements` section?

**Review by mfshao — APPROVED** [review]
👍

**Pull request merged**
